src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task01.java

```diff
@@ -40,6 +40,6 @@ public static void main(String[] args) {
     static boolean isTopPlayer(int sizeTop, int positionPlayerByDamage) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return false;
+        return sizeTop>=positionPlayerByDamage;
```

Comment:
Violations of the Code Convention!

Reply:
fix: tasks 1 for theme 'conditional-statements'